src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -98,6 +98,16 @@ protected ActionResult execute() {
                                                         new Text(emailContent),
                                                         null);
             data.emailToEdit.emailId = emailId;
+        } else {
+            if (addressModeOn) {
+                statusToAdmin = "Email sent to " + addressReceiverListString;
+                statusToUser.add(new StatusMessage("Email sent to "
+                        + addressReceiverListString, StatusMessageColor.SUCCESS));
```

Comment:
Sorry, best to leave it best way. Looking more closely, the surrounding code in that file also does it this way; I think its to avoid problems caused by the statusToAdmin variable changing further down in the code, therefore causing the status message to change, since it doesn't do a deep copy of the string during construction.